internal/server/data/migrations.go

```diff
@@ -597,3 +598,19 @@ func setDefaultOrgID() *migrator.Migration {
 		},
 	}
 }
+
+func addIdentityVerifiedFields() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "2022-09-01T15:00",
+		Migrate: func(tx migrator.DB) error {
+			stmt := `
+ALTER TABLE identities
+	ADD COLUMN IF NOT EXISTS verified boolean NOT NULL DEFAULT false,
+	ADD COLUMN IF NOT EXISTS verification_token text NOT NULL DEFAULT substr(replace(translate(encode(decode(MD5(random()::text), 'hex'),'base64'),'/+','=='),'=',''), 1, 10);
```

Comment:
How does postgres handle duplicates if this happens to generate a random token that already existed?

Reply:
💥 
though the chances of that are about in in 61^10, which is so absurdly large we don't need to worry about it.